reconfigure/http.js

```diff
@@ -25,7 +25,10 @@ Reconfigure.prototype.register = cadence(function (async, post) {
     async(function () {
         this._coordinator.listen(post.body.url, async())
     }, function () {
-        return 'listener ' + post.body.url + ' has joined'
+        return {
+            response: 'listener ' + post.body.url + ' has joined',
```

Comment:
Let's do this; `extant` is a boolean indicating whether or not the value already existing. `url` is a repeat of the url that was submitted, and `success` indicates that nothing bad happened.